templates/resources/grant_ownership.md.tmpl

```diff
@@ -22,8 +22,6 @@ description: |-
 ## Example Usage
 
 {{ tffile (printf "examples/resources/%s/resource.tf" .Name)}}
-
--> **Note** If you don't manage database role in Terraform, you can construct the proper id yourself, consult [identifiers guide](../guides/identifiers#new-computed-fully-qualified-name-field-in-resources)
```

Comment:
You removed this message, but didn't include the common one (in the resulting md file it's removed), I think it would be the best to extract the common message to some kind of `.md` file and include it with `codefile` (in specified templates as this one and in the common resources.md.tmpl) as Artur suggested, because right now this message is not available for the resources for which we have custom templates.

Reply:
Added to every resource.